examples/templates/example.html

```diff
@@ -108,7 +108,14 @@
       <div class="row-fluid">
         <a class="codepen-button pull-right"><i class="fa fa-codepen"></i> Edit</a>
         <div class="span12">
-          <h4 id="title">{{ title }}</h4>
+          <h4 id="title">
+            {{ title }}
+          </h4>
+          <p class="tags">
+            {{#each tags}}
+              <a href="./index.html?q={{.}}"><span class="label label-primary">{{.}}</span></a>
```

Comment:
Using `label` classes on the anchor directly gives an even nicer clickable tag :)
```suggestion
              <a href="./index.html?q={{.}}" class="label label-primary">{{.}}</a>
```